.circleci/config.yml

```diff
@@ -181,6 +181,7 @@ jobs:
           environment:
             EXECUTION_TYPE: apiSmoke
             PROCESS: dev
+            APPOINTMENT_FLAG: ${APPOINTMENT_FLAG}
```

Comment:
where does this value come from? circle ci context?

Reply:
circle ci yes